examples/fancy-javascript/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "gastby-fancy-javascript",
+  "version": "1.0.0",
+  "description": "personal site and blog",
+  "main": "index.js",
+  "scripts": {
+    "develop": "node_modules/.bin/gatsby develop",
+    "dev:hard": "rm -rf .cache && rm -rf public && npm run develop",
+    "build": "npm run clean & .\\node_modules\\.bin\\gatsby build",
+    "serve-build": "node_modules/.bin/gatsby serve-build",
+    "clean:public": "rm -rf public & mkdir public",
+    "clean": "npm run clean:public",
+    "lint": "./node_modules/.bin/eslint --ext .js,.jsx --ignore-pattern public .",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "personal",
+    "blog"
+  ],
+  "author": "jbolda",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby/issues"
+  },
+  "homepage": "https://www.gatsbyjs.org",
+  "dependencies": {
+    "bulma": "0.4.2",
+    "d3": "4.9.1",
+    "gatsby": "^1.1.0",
+    "gatsby-link": "^1.1.0",
```

Comment:
Many of these plugins don't have a 1.1.0 release yet — Lerna doesn't make a new release for a package unless there's been changes.